cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -882,7 +885,17 @@ pub(crate) fn emit<O: MachSectionOutput>(inst: &Inst, sink: &mut O) {
                 }
             }
         }
-
+        Inst::SSE_Scalar_Mov_R_R { is_64, src, dst } => {
```

Comment:
Errr, this is tricky, for reasons to do with copy coalescing in regalloc.  At least for now, I'd prefer to have an Xmm-to-Xmm move instruction that operates only on the entire register.  We can come back and refine this later.  So you'd drop the `Scalar` bit here, giving `SSE_Mov_R_R`.  And correspondingly emit whatever Intel recommends for Xmm-to-Xmm moves .. is it `movapd reg, reg` perhaps?